packages/debug-nodejs/src/node/debug-nodejs.ts

```diff
@@ -29,33 +34,53 @@ export class NodeJsDebugAdapterContribution implements DebugAdapterContribution
     provideDebugConfigurations = [{
         type: this.debugType,
         request: 'attach',
-        name: 'Attach by PID',
-        processId: ''
+        name: 'Debug (Attach)',
+        processId: '<pid>'
```

Comment:
It is often easier to edit when we leave an empty string, instead of `<pid>`.
It is already named `processId` anyway.